obspy/io/seiscomp/inventory.py

```diff
@@ -143,7 +143,8 @@ def _ns(tagname):
     responses = {
         "responsePAZ": {},
         "responsePolynomial": {},
-        "responseFIR": {}
+        "responseFIR": {},
+        "responseIIR": {}
```

Comment:
You could later on do `responses.setdefault(response_type, {})[public_id] = response_element` and not worry about setting up all potential response types ahead of time?

Reply:
I have considered this suggestion. However I have decided not to action this any further. I would rather that the responses list was not explicit like this, but the previous code also only looked for specific hard-wired response types, and with the current changes at least this list of response types is centralized in one code statement. Using `setdefault` is a good way to simplify population of a dictionary, however making such a change here would result in losing the check for duplication of the public_id, which raises a potentially informative exception.

Reply:
Fine with me, didn't look at all implementation details, was just thinking about what happens when an unknown response type is encountered, but I guess you got all types covered that can be encountered.